pkg/kubelet/volumemanager/reconciler/reconciler.go

```diff
@@ -208,7 +208,7 @@ func (rc *reconciler) mountAttachVolumes() {
 			if rc.controllerAttachDetachEnabled || !volumeToMount.PluginIsAttachable {
 				// Volume is not attached (or doesn't implement attacher), kubelet attach is disabled, wait
 				// for controller to finish attaching volume.
-				klog.V(5).InfoS(volumeToMount.GenerateMsgDetailed("Starting operationExecutor.VerifyControllerAttachedVolume", ""))
+				klog.V(5).InfoS(volumeToMount.GenerateMsgDetailed("Starting operationExecutor.VerifyControllerAttachedVolume", ""), klog.KObj(volumeToMount.Pod))
```

Comment:
Can you show an example of what this log looks like now?

`GenerateMsgDetailed` also contains pod information. Maybe we need to clean that up too.

Reply:
Not from this but from a few logs below

```
{"ts":1624886288131.6914,"caller":"reconciler/reconciler.go:254","msg":"Starting operationExecutor.MountVolume for volume \"usr-local-share-ca-certificates\" (UniqueName: \"kubernetes.io/host-path/25e80667e06cd3d268ce52026676a8a1-usr-local-share-ca-certificates\") pod \"kube-apiserver-kind-control-plane\" (UID: \"25e80667e06cd3d268ce52026676a8a1\") ","v":4,"pod":"kube-system/kube-apiserver-kind-control-plane"}
```

We could in theory remove the pod information from `GenerateMsgDetailed` to get less duplication, though that doesn't seem to be critical. The important bit is the structured context being added.